dashboard/src/proxy.ts

```diff
@@ -200,13 +72,40 @@ export class ProxyDashboard {
     // 2. Disable inspection if user moved away from inspect tab
     // 3. Do nothing if activeTabId == clickedTabId
     if (clickedTabId !== activeTabId) {
-      if (clickedTabId === 'proxyInspect') {
-        this.websocketApi.enableInspection()
-      } else if (activeTabId === 'proxyInspect') {
-        this.websocketApi.disableInspection()
+      //deactivateAll
+      this.home.disable();
+      this.apiDevelopment.disable();
+      this.websocketApi.disable();  
+      this.shortLinks.disable();
+      this.controls.disable();
+      this.settings.disable();
```

Comment:
How about just calling `$('#app>app-tab').hide()`.  Then we at least won't need to add / remove new `this.newtab.disable()` when tabs are added / removed.